index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    
+    <div class="container">
+        <nav class="mobile-nav">
+            <div class="shapes">
+                <div class="images">
+                    <img id="triangleImage" src="images/ShapeTriangle.png" alt="triangel shape">
+                    <img id="rectangleImage" src="images/Shape.png" alt="rectangle shape">
+
+                </div>
+            </div>
+            <div class="hamburger">
+                <span class="rec rectangle1"></span>
+                <span class="rec rectangle2"></span>
+                <span class="rec rectangle3"></span>
+            </div>
+        </nav>
+        <div class="main">
+            <div class="header">Ayesha Arshad</div>
+            <div class="discription">Hello! I am a software developer! I can help you build a product, feature or
+                website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.</div>
```

Comment:
- [ ] And here as you can see it would be more organized if you use the `p` tag instead of `div` because there is `p` tag is more convenient in this case because it is a block element used to designate a paragraph. It is useful because Internet browsers add a bit of margin before after each element. The following sections contain information about this tag.